src/ontology/hp.Makefile

```diff
@@ -426,3 +426,20 @@ help:
 	@echo "$$data"
 	echo "Migrating EQs from MP to HP:"
 	echo "make ADOPT_EQS_MAPPING_URL=SOMEURL migrate_eqs_to_edit"
+
+.PHONY: diff
+diff: diff-table diff-kgcl diff-yaml
+
+tmp/hp-released.obo:
+	wget http://purl.obolibrary.org/obo/hp.obo -O $@ && cp ../../hp-full.obo tmp/hp-full.obo
+
+diff-table: tmp/hp-released.obo tmp/hp-full.obo
```

Comment:
Diff should be against hp-base.owl (for sure hp-base, and using OBO format is problematic).